res/xml/qwerty_vi.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
+    <key key0="q" key1="~" key2="1" key4="esc"/>
+    <key key0="w" key1="loc accent_breve" key2="2" key3="\@" key4="accent_horn"/>
```

Comment:
Breve is needed ? It shouldn't have the `loc` prefix then. The `loc` prefix is used to place a key but to show it only if it is an "extra key" or is enabled in the settings (eg. like `capslock`)

Reply:
oh, I forgor again 💀
thanks for figure it out
(it's 4:53 AM now and I haven't sleep yet)

Reply:
Removed in latest commit